### PR TITLE
updated README.md by following ros2_documentation README.md

### PR DESCRIPTION
**Pull request by razr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,42 @@
-# rtwg_documentation
+# ROS 2 Real-Time Working Group Documentation
 
 [![Documentation Status](https://readthedocs.org/projects/real-time-working-group/badge/?version=latest)](https://real-time-working-group.readthedocs.io/en/latest/?badge=latest)
 
-## On-line documentation link
+This repository contains the sources for the ROS 2 real-time working group documentation that is hosted at https://real-time-working-group.readthedocs.io.
+The sources from this repository are built and uploaded to the site manually.
```
Comment (LanderU) [inline README.md:6]: There is a hook that automatically fetch and build the latest version of the master branch on every commit.

**Review by LanderU — CHANGES_REQUESTED** [review]

**Review by razr — COMMENTED** [review]
thanks for fixing it

File: README.md [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+Contributions to this site are most welcome.
+Please be sure to read the below sections carefully before contributing.
 
+The site is built using [Sphinx](https://www.sphinx-doc.org/)
+
+### Branch structure
+
+This repository is setup with one branch per ROS 2 distribution to handle differences between the distributions.
+If a change is common to all ROS 2 distributions, it should be made to the `rolling` branch (and then will be backported as appropriate).
+If a change is specific to a particular ROS 2 distribution, it should be made to the respective branch.
```
Comment (christophebedard) [inline README.md:19]: does the section and the lines above really apply? We're only using `master` for now.

Comment (razr) [reply README.md:19]: @christophebedard - this is what we have discussed last week.  We need to change the branch name ```master``` to ```rolling``` or ```foxy```

**Comment by LanderU** [comment]
@razr, is it ready for a second review?

**Review by LanderU — APPROVED** [review]
LGTM!

I'll create the branch architecture.

**Pull request merged**